sonarqubeandjacoco/pom.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Instead of putting this module at the root of the repo, let's put it in the testing-modules directory and call it "sonarqube-jacoco".